tools/run_tests/dockerize/build_docker_and_run_tests.sh

```diff
@@ -82,6 +82,7 @@ docker run \
   -e "BUILD_URL=$BUILD_URL" \
   -e "JOB_BASE_NAME=$JOB_BASE_NAME" \
   -i $TTY_FLAG \
+  --sysctl net.ipv6.conf.all.disable_ipv6=0 \
```

Comment:
```
sudo sed -i s/DOCKER_OPTS=/DOCKER_OPTS=\"--ipv6\"/g /etc/init.d/docker
sudo /etc/init.d/docker restart
```
Above isn't necessary (at least for running sanitizers). Only the `--sysctl net.ipv6.conf.all.disable_ipv6=0` flag is necessary.